File: cli/parser/testdata/python/basic.py
# Basic xschema calls - all should be parsed
from xschema import create_xschema_client
from xschema_pydantic import pydantic_adapter

xschema = create_xschema_client({})

# from_url with double quotes
xschema.from_url("User", "https://api.example.com/user.json", pydantic_adapter)

# from_file with double quotes
xschema.from_file("Post", "./schemas/post.json", pydantic_adapter)

File: cli/parser/testdata/typescript/basic.ts
// Basic xschema calls - all should be parsed
import { createXSchemaClient } from "@xschema/client";
import { zodAdapter } from "@xschema/zod";

const xschema = createXSchemaClient({});

// fromURL with double quotes
xschema.fromURL("User", "https://api.example.com/user.json", zodAdapter);

// fromFile with double quotes
xschema.fromFile("Post", "./schemas/post.json", zodAdapter);
